force-app/main/default/lwc/contactManagerModal/contactManagerModal.js

```diff
@@ -0,0 +1,99 @@
+import { api } from "lwc";
+import LightningModal from "lightning/modal";
+import getRegionManagerByUserId from "@salesforce/apex/RegionManagerController.getRegionManagerByUserId";
+import contactManager from "@salesforce/apex/RegionManagerController.contactManager";
+import Id from "@salesforce/user/Id";
+
+const FORMATS = [
+  "font",
+  "size",
+  "bold",
+  "italic",
+  "underline",
+  "strike",
+  "list",
+  "indent",
+  "align",
+  "link",
+  "clean",
+  "table",
+  "header"
+];
+
+export default class ContactManagerModal extends LightningModal {
+  @api content;
+  contactData = {
+    managerEmail: "",
+    message: "",
+    subject: "",
+    userEmail: ""
+  };
+  formats = FORMATS;
+  validity = true;
+
+  connectedCallback() {
+    this.fetchManagerEmail();
+  }
+
+  fetchManagerEmail() {
+    getRegionManagerByUserId({ userId: Id })
```

Comment:
you don't really need the `fetchManagerEmail` as a separate function as it only triggers another function, you could place the `getRegionManagerByUserId` right inside the conenctedCallback